dm/pkg/checker/binlog_test.go

```diff
@@ -127,3 +127,75 @@ func TestBinlogDB(t *testing.T) {
 		require.Equal(t, cs.state, r.State)
 	}
 }
+
+func TestMySQLBinlogRowImageChecker(t *testing.T) {
+	db, mock, err := sqlmock.New()
+	require.Nil(t, err)
+	ctx := context.Background()
+
+	cases := []struct {
+		version   string
+		state     State
+		needCheck bool
+		rowImage  string
+	}{
+		// mysql < 5.6.2 don't need check
+		{
+			version:   "5.6.1-log",
+			state:     StateSuccess,
+			needCheck: false,
+			rowImage:  "full",
```

Comment:
If need not check, we maybe should test `rowImage != "full"`.

Reply:
It's been a long time, we'll be waiting for you to come back soon 🤣

Reply:
I thought again, for the no need check scenario there is no `binlog_row_image` variable in the db, so this test scenario is redundant? (It doesn't happen in the real world)

Reply:
needCheck will make sure these's no mock result for SELECT binlog_row_image. In those cases I think remove `rowImage:  "full",` should also be OK